wp-seo.php

```diff
@@ -49,6 +58,6 @@
 	require_once ( WP_SEO_PATH . '/php/class-wp-seo.php' );
 
 	// Common helpers and miscellaneous functions.
-	require_once( WP_SEO_PATH . '/php/functions.php' );
+	require_once ( WP_SEO_PATH . '/php/functions.php' );
```

Comment:
Is that coding standards, have a space between `require_once` and the parens? In other words, is it supposed to be treated like a control structure instead of a function?